pkgs/native_assets_cli/lib/src/data_assets/validation.dart

```diff
@@ -76,6 +91,26 @@ void _validateDataAsset(
   errors.addAll(_validateFile('Data asset ${dataAsset.name} file', file));
 }
 
+List<String> _validateDataAssetSyntax(EncodedAsset encodedAsset) {
```

Comment:
Documentation would be much appreciated ;) `List<String>` is not a very expressive return type.

Reply:
Ah, we actually have `typedef ValidationErrors`. Good catch!